models/issue_comment.go

```diff
@@ -694,6 +700,17 @@ func createDeadlineComment(e *xorm.Session, doer *User, issue *Issue, newDeadlin
 	})
 }
 
+func createChangePullRequestTargetBranchComment(e *xorm.Session, doer *User, repo *Repository, issue *Issue, oldBranch string, newBranch string) (*Comment, error) {
```

Comment:
No need for this function, just use createComment directly

Reply:
So that was changed with the refactoring of the notification system. All right, will fix that later today. :)